code/game/gamemodes/wizard/artefact.dm

```diff
@@ -213,3 +213,383 @@
+				M.equip_to_slot_or_del(new /obj/item/device/radio/headset(M), slot_l_ear)
+				M.equip_to_slot_or_del(sword, slot_r_hand)
+
+			if("highlander")
+				M.equip_to_slot_or_del(new /obj/item/clothing/under/kilt(M), slot_w_uniform)
+				M.equip_to_slot_or_del(new /obj/item/device/radio/headset(M), slot_l_ear)
+				M.equip_to_slot_or_del(new /obj/item/clothing/head/beret(M), slot_head)
+				M.equip_to_slot_or_del(new /obj/item/clothing/shoes/combat(M), slot_shoes)
+				M.equip_to_slot_or_del(sword, slot_r_hand)
+
+			if("clown")
+				M.equip_to_slot_or_del(new /obj/item/clothing/under/rank/clown(M), slot_w_uniform)
+				M.equip_to_slot_or_del(new /obj/item/device/radio/headset(M), slot_l_ear)
+				M.equip_to_slot_or_del(new /obj/item/clothing/shoes/clown_shoes(M), slot_shoes)
+				M.equip_to_slot_or_del(new /obj/item/clothing/mask/gas/clown_hat(M), slot_wear_mask)
+				M.equip_to_slot_or_del(new /obj/item/weapon/bikehorn(M), slot_l_store)
+				M.equip_to_slot_or_del(sword, slot_r_hand)
+
+			if("killer")
+				M.equip_to_slot_or_del(new /obj/item/clothing/under/overalls(M), slot_w_uniform)
+				M.equip_to_slot_or_del(new /obj/item/clothing/shoes/white(M), slot_shoes)
+				M.equip_to_slot_or_del(new /obj/item/clothing/gloves/color/latex(M), slot_gloves)
+				M.equip_to_slot_or_del(new /obj/item/clothing/mask/surgical(M), slot_wear_mask)
+				M.equip_to_slot_or_del(new /obj/item/clothing/head/welding(M), slot_head)
+				M.equip_to_slot_or_del(new /obj/item/device/radio/headset(M), slot_l_ear)
+				M.equip_to_slot_or_del(new /obj/item/clothing/suit/apron(M), slot_wear_suit)
+				M.equip_to_slot_or_del(new /obj/item/weapon/kitchen/utensil/knife(M), slot_l_store)
+				M.equip_to_slot_or_del(new /obj/item/weapon/scalpel(M), slot_r_store)
+				M.equip_to_slot_or_del(sword, slot_r_hand)
+				for(var/obj/item/carried_item in M.contents)
+					if(!istype(carried_item, /obj/item/weapon/implant))
+						carried_item.add_blood(M)
+
+			if("pirate")
+				M.equip_to_slot_or_del(new /obj/item/clothing/under/pirate(M), slot_w_uniform)
+				M.equip_to_slot_or_del(new /obj/item/clothing/shoes/brown(M), slot_shoes)
+				M.equip_to_slot_or_del(new /obj/item/clothing/head/bandana(M), slot_head)
+				M.equip_to_slot_or_del(new /obj/item/clothing/glasses/eyepatch(M), slot_glasses)
+				M.equip_to_slot_or_del(new /obj/item/device/radio/headset(M), slot_l_ear)
+				M.equip_to_slot_or_del(sword, slot_r_hand)
+
+			if("soviet")
+				M.equip_to_slot_or_del(new /obj/item/clothing/head/hgpiratecap(M), slot_head)
+				M.equip_to_slot_or_del(new /obj/item/clothing/shoes/combat(M), slot_shoes)
+				M.equip_to_slot_or_del(new /obj/item/clothing/gloves/combat(M), slot_gloves)
+				M.equip_to_slot_or_del(new /obj/item/device/radio/headset(M), slot_l_ear)
+				M.equip_to_slot_or_del(new /obj/item/clothing/suit/hgpirate(M), slot_wear_suit)
+				M.equip_to_slot_or_del(new /obj/item/clothing/under/soviet(M), slot_w_uniform)
+				M.equip_to_slot_or_del(sword, slot_r_hand)
```

Comment:
In this case, they need to be del, or it's going to wind up doing weird things if you happen to roll for golem, for instance.